package.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "soundzcape",
+    "version": "0.1.0",
+    "description": "2D platformer with procedural wave map.",
+    "author": "KryptonChicken",
+    "license": "Apache-2.0",
+    "scripts": {
+        "build": "webpack"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/tommyip/soundzcape.git"
```

Comment:
Wouldn't it be better to use `https://github.com/KryptonChicken/soundzcape.git` ?